master/internal/rm/agentrm/agent_state.go

```diff
@@ -44,8 +43,8 @@ type slot struct {
 type agentState struct {
 	syslog *log.Entry
 
-	// Handler is agent actor reference.
-	Handler          *actor.Ref
+	id               agentID // TODO(!!!): Why agentID and aproto.ID? Let's just have one or the other.
```

Comment:
question: what is agentID/aproto.ID?

Reply:
IDK, two types for the same thing. I plan to follow up remove one of them.